spec/system/meetings/user_create_meeting_spec.rb

```diff
@@ -21,10 +21,13 @@
     expect(page).to have_content 'Reunião: Daily'
     expect(page).to have_content "Autor\n#{user.email}"
     expect(page).to have_content "Descrição\nSem descrição"
-    expect(page).to have_content "Data e horário\n24/05/2024, 14:00"
+    expect(page).to have_content "Data e horário\n24/11/2024, 14:00"
     expect(page).to have_content "Duração\n2h"
     expect(page).to have_content "Endereço\nhttps://meet.google.com"
     expect(page).to have_link 'https://meet.google.com'
+    expect do
+      NotifyParticipantsJob.perform_later(Meeting.last)
```

Comment:
Pelo o que entendi, a lógica dos testes de sistema é garantir que seu Job está sendo enfileirado internamente na execução do código. Pra isso você só precisa criar um espião do Job e testar se ele foi chamado dentro do seu código, nesse caso, na validação do model. Então não faz sentido você enfileirar ele dentro do teste. Meio que você estaria testando se é verdade algo que você forçou a ser verdade.

Reply:
Iremos implementar sua sugestão, quando fizemos esse teste estávamos começando a entender o fluxo do jobs nos testes rsrs. Obrigado por nos avisar sobre isso!